core/src/main/java/com/microsoft/applicationinsights/internal/channel/common/TransmissionSendResult.java

```diff
@@ -24,7 +24,7 @@
 /**
  * Created by gupele on 2/10/2015.
  */
-enum TransmissionSendResult {
+public enum TransmissionSendResult {
```

Comment:
Why do we need to make this enum public? Is it required for test cases? If yes please follow the Java Doc requirements and update the comment to be java doc comment with description.

Reply:
Haha, got it!